packages/tab-indicator/index.scss

```diff
@@ -1 +1,6 @@
 @import "@material/tab-indicator/mdc-tab-indicator";
+
+.mdc-tab-indicator__content--icon .material-icons {
```

Comment:
dont do this. Instead make the material-icons stylesheet load before the mdc-tab-indicator stylesheet

Reply:
Actually the `.material-icons` class has the font-size: 24px. So rearranging the stylesheets will not help here.

I moved the logic around so that we now have the ref on the TabIndicatorElement. We then use .getElementsByClassName, which I think is fine in this case. The call is scoped to just the tabIndicator.